src/pages/components/notification/usage.mdx

```diff
@@ -4,79 +4,166 @@ description: Notifications are messages that communicate information to the user
+
+## Content
+
+Notifications provide limited space for content, and therefore the content must be concise. A user should be able to quickly scan the notification, be apprised of the situation, and know what to do next.
+
+### Title
+
+- The title should be short and descriptive, explaining the most important piece of information.
+- When possible, communicate the main message using just the title.
+- Use a period only if the title is a full sentence.
 
-Inline notifications appear near its related item. In [forms](/components/form/code), we recommend placing the inline notification at the bottom of the form, right before the submission buttons. Depending on the context of the page, inline notifications can appear above the content as well.
+### Body content
+
+- The body of the notification should be no more than two lines.
+- Be concise and avoid repeating or paraphrasing the title.
+- Explain how to resolve the issue by including any troubleshooting actions or next steps. You can include links within the notification body that redirect the user to next steps.
+
+### Action
+
+- Keep labels consice and clearly indicate the action the user can take.
+- Limit action labels to two words or less.
+
+### Timestamp
+
+- The timestamp reflects the time the toast notification was sent. Timestamps should be localized to the user's timezone or preferences.
+
+For further content guidance, see Carbon's [content guidelines](/guidelines/content/general).
+
+## Modifiers
+
+### High and low contrast
+
+Carbon supports high and low constrast style notifications. High-contrast notifications are best for critical messaginging while low-contrast notifications are less visually disruptive for users.
+
+It's' up to the product team to decide which notification style to use in their product. Inline and toast notifications can use different styles but you should never mix styles within each notification type. When in doubt, use low-contrast notifications.
 
 <Row>
 <Column colLg={8}>
 
-![Form example with inline notification](images/notification-usage-3.png)
+![High and low contrast inline notifications](images/notification-usage-4.png)
 
 </Column>
 </Row>
+
+### Action
+
+**Inline notification**
+
+Inline notifications have an optional ghost button action that is right aligned. This button should allow users to take further action on the notification.
+
+**Toast notifications**
+
+Toast notifications can include a link at the end of their body content. This link should be short and navigate users to a page or modal where they can take action to address the notification or find further information. Because toast notifications automatically dismiss, it is important that there are alternative routes to navigate to the link destination.
```

Comment:
> Because toast notifications automatically dismiss, **it is important** that there are alternative routes to navigate to the link destination.

Should we potentially include stronger language around this in the sense that they must provide an alternate way to view the notification?